game_server/Controllers/PlayGameController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.EntityFrameworkCore;
+using game_server.Models;
+
+namespace game_server.Controllers
+{
+
+  [ApiController]
+  [Route("[controller]")]
+  public class PlayGameController : Controller
+  {
+    private readonly ILogger<PlayGameController> _logger;
+    private GameSessionModelDBContext _context;
+    private games.ICardGame _cardGameService;
+
+    public PlayGameController(
+      ILogger<PlayGameController> logger,
+      GameSessionModelDBContext context,
+      games.ICardGame cardGame)
+    {
+        _logger = logger;
+        _context = context;
+        _cardGameService = cardGame;
+    }
+
+    [EnableCors]
+    [HttpPost("GetValidPlays/{sessionid}")]
+    async public Task<ValidPlaysPayload> GetValidPlays([FromBody]ValidPlaysIncomingPayload incomingPayload){
+      Console.WriteLine(incomingPayload);
+      // TODO - Specific gamename resolution
+      var escoba_info = _context.Games.Where(x => x.GameName == "escoba").SingleOrDefault();
```

Comment:
given `sessionid` as input the gameinfo `escoba_info` here can be retrieved from the `gameSession` once looked up